# Publishes the current state of the kalman filter, intended for debug purposes

float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
float64 sl
float64 sa


